website/docs/docs/build/packages.md

```diff
@@ -366,3 +372,18 @@ packages:
 ```
 
 </File>
+
+### About dependencies.yml
+
+There are some important differences between using a `dependencies.yml` compared to a `packages.yml` file:
```

Comment:
Do we still need this clarification section? It feels duplicative with the `:::info` block callout above